DashAI/back/core/runner.py

```diff
@@ -69,8 +66,8 @@ def execute_run(
             model_metrics = {
                 split: {
                     metric.__name__: metric.score(
-                        formated_dataset[split]["output"],
-                        model.predict(formated_dataset[split]["input"]),
+                        dataset[split],
+                        model.predict(dataset[split]),
```

Comment:
Why not to predict just one time and then use this values to calculate the metric?
Here you are doing the same prediction the number of metrics associated with the task.

Reply:
Also, why here you use `dataset` directly and not `prepared_dataset`?